### PR TITLE
SOC-1103 | Human tracking on UserSignup

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/SOC-1103

We'd like to understand better how many pageviews are generated by bots on UserSignup pages.
This simple code attaches to few events which we believe are not usually fired by bots, then if event is detected we track an event in our analytics systems. User can generate only one event per pageview.

### CODE REVIEW

File: extensions/wikia/UserLogin/js/UserSignup.js [hunk 1]
```diff
@@ -43,6 +44,29 @@
...
+
+		/**
+		 * We'd like to understand better how many pageviews are generated by bots on UserSignup pages.
+		 * This simple code attaches to few events which we believe are not usually fired by bots,
+		 * then if event is detected we track an event in our analytics systems.
+		 */
+		setupHumanTracking: function () {
+			$(document).on(
+				'mousemove.humanTracking keypress.humanTracking touchstart.humanTracking',
+				function(event) {
```
Comment (bkoval) [inline extensions/wikia/UserLogin/js/UserSignup.js]: Space after "function" keyword

File: extensions/wikia/UserLogin/js/UserSignup.js [hunk 2]
```diff
@@ -43,6 +44,29 @@
...
 				.on('change.UserSignup', this.validator.validateBirthdate.bind(this.validator));
+		},
+
+		/**
+		 * We'd like to understand better how many pageviews are generated by bots on UserSignup pages.
+		 * This simple code attaches to few events which we believe are not usually fired by bots,
+		 * then if event is detected we track an event in our analytics systems.
+		 */
+		setupHumanTracking: function () {
+			$(document).on(
```
Comment (bkoval) [inline extensions/wikia/UserLogin/js/UserSignup.js:56]: Is there any way to flatten the callback chain here? :) Defining callbacks at the top of this method might be a good start :)

Comment (kvas-damian) [inline extensions/wikia/UserLogin/js/UserSignup.js:56]: done

**Comment by bkoval** [comment]
LGTM :+1:

**Comment by garthwebb** [comment]
LGTM

**Pull request merged**